### PR TITLE
Update Gemfile.lock

### PR DESCRIPTION
**Pull request by agrare**

### CODE REVIEW

**Comment by NickLaMuro** [comment]
@agrare Could we also update `thin` to 1.8.0?  I have run into this issue more times than I can remember, and it gets me every time:

- https://github.com/macournoyer/thin/issues/365
- https://github.com/macournoyer/thin/pull/364

We can do it as a separate PR as well if people prefer.

File: Gemfile.lock [hunk 1]
```diff
@@ -360,25 +354,26 @@ GEM
     sync (0.5.0)
     sysexits (1.2.0)
     temple (0.8.2)
-    thin (1.7.2)
+    thin (1.8.1)
```
Comment (agrare) [inline Gemfile.lock:357]: Absolutely @NickLaMuro :point_up:

Comment (NickLaMuro) [reply Gemfile.lock:357]: Oh!  You did a generic `bundle update`, not just for celluloid.  Cool beans.  Thanks!

**Review by NickLaMuro — APPROVED** [review]
Looks good to me, but I will let someone else merged like @bdunne or @Fryguy

File: Gemfile.lock (hunk 1 above)
Comment (agrare) [reply Gemfile.lock:357]: Yeah celluloid was the only one which was failing on the latest version so I limited it to the prior version.

File: spec/models/batch_job_spec.rb [hunk 2]
```diff
@@ -85,8 +85,9 @@
 
   describe "#check_complete" do
     it "when destroyed by another checker" do
-      job = described_class.create!.tap(&:destroy)
+      job = described_class.new
```
Comment (bdunne) [inline spec/models/batch_job_spec.rb:88]: How is this related to rspec mocks?

Comment (agrare) [reply spec/models/batch_job_spec.rb:88]: Because a destroyed object is frozen, and rspec 3.10 raises an error about mocking a frozen object (see the error referenced here https://github.com/ManageIQ/miq_bot/pull/566#issuecomment-904082377)

Comment (bdunne) [reply spec/models/batch_job_spec.rb:88]: So, should we set up the expectation before destroying the record instead?

File: spec/models/batch_job_spec.rb [hunk 3]
```diff
@@ -85,8 +85,9 @@
...
 
+      allow(job).to receive(:reload).and_raise(ActiveRecord::RecordNotFound)
       expect(job).to_not receive(:finalize!)
 
       job.check_complete
```
Comment (bdunne) [inline spec/models/batch_job_spec.rb:93]:
```suggestion
      job = described_class.create!

      expect(job).to_not receive(:finalize!)

       job.destroy
      
      job.check_complete
```

File: spec/models/batch_job_spec.rb (hunk 2 above)
Comment (agrare) [reply spec/models/batch_job_spec.rb:88]: Sure

File: Gemfile.lock (hunk 1 above)
Comment (kbrock) [reply Gemfile.lock:357]: celluloid was all the rage for a while but died out after sidekick rewrote to not include it.
TBH, kinda surprised to see it.
